spec/cb/utils/validator_spec.rb

```diff
@@ -14,125 +14,138 @@ module Cb
   describe ResponseValidator do
     let(:response)          { double(HTTParty::Response) }
     let(:response_property) { double('HTTParty::Response.response') }
+    let(:response_body)     { nil }
+    let(:response_code)     { 200 }
 
     before :each do
       allow(response).to receive(:response).and_return(response_property)
-      allow(response).to receive(:code).and_return(200)
+      allow(response).to receive(:code).and_return(response_code)
+      allow(response_property).to receive(:body).and_return(response_body)
     end
 
-    it 'should return empty hash when body is empty' do
-      allow(response.response).to receive(:body).and_return('')
-      validation = ResponseValidator.validate(response)
-      expect(validation.empty?).to be_truthy
-    end
+    context 'with an empty body' do
+      let(:response_body) { '' }
 
-    it 'should return empty hash when body is improper json/xml' do
-      allow(response.response).to receive(:body).and_return('json')
-      validation = ResponseValidator.validate(response)
-      expect(validation.empty?).to be_truthy
+      it 'returns an empty hash' do
+        validation = ResponseValidator.validate(response)
+        expect(validation).to be_empty
```

Comment:
I don't think we should be caring so much about classes. Both for communist reasons, and because Ruby is dynamically typed. We care that it responds to the appropriate method calls